wagtail/admin/templates/wagtailadmin/shared/page_status_tag_new.html

```diff
@@ -0,0 +1,55 @@
+{% load i18n wagtailadmin_tags %}
+{% comment %}
+
+    Variables accepted by this template:
+
+    - `page` - A wagtail page object
+    - `classes` - String of extra css classes to pass to this component
+{% endcomment %}
+
+{% test_page_is_public page as is_public %}
+
+{% if page.live and page.url is not None %}
+    <a href="{{ page.url }}" target="_blank" rel="noreferrer"
+        class="
+            w-inline-flex
+            w-items-center
+            w-justify-center
+            w-whitespace-nowrap
+            w-px-1
+            w-ml-3
+            w-text-[11px]
+            w-rounded-sm
+            w-bg-transparent
+            w-text-grey-400
+            w-border
+            w-border-grey-100
+            w-no-underline
+            w-font-semibold
+            hover:w-border-primary
+            hover:w-text-primary
+            w-transition
+            {{ classes }}"
+        style="text-transform: uppercase;" {# We use inline styles here as a special case to limit the amount uppercase is used with css classes #}
```

Comment:
Could you use Tailwind’s `[text-transform:_uppercase]` instead?

Reply:
Hmm couldn't get this to work so i created a css class now 😭